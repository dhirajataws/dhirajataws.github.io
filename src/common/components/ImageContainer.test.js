import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { ImageContainer } from './ImageContainer';
test('ImageContainer components ', () => {
  const tree = renderer.create(<ImageContainer xs="12" />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('display', 'flex')
  expect(tree).toHaveStyleRule('position', 'relative')
  expect(tree).toHaveStyleRule('margin', '8vw')
  // expect(tree).toHaveStyleRule('width', '100%', {
  //   media: '(min-width:320px)',
  // }) TODO revisit
})