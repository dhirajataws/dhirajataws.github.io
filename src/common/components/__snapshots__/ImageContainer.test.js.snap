// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageContainer components  1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
  margin: 8vw;
}

@media only screen and (min-width:320px) {
  .c0 {
    width: 100%;
  }
}

<div
  className="c0"
/>
`;
